includes/kaltura_derivatives.inc

```diff
@@ -68,10 +68,21 @@ function islandora_video_streaming_kaltura_upload($object, $force = false) {
 
       $entity = kaltura_entry_create(array('kaltura_entryid' => $entry->id));
       kaltura_map_properties($entry, $entity);
-
-      kaltura_save_entry_metadata($entity);
+      $entity->kaltura_title = $object->id; 
+      $entity->kaltura_description = $object->label;
+      
       kaltura_entry_save($entity); //write entity to drupal db
-
+      
+      try {
+        $kaltura_baseEntry = new KalturaBaseEntry();
+        $kaltura_baseEntry->name = $object->id;
+        $kaltura_baseEntry->description = $object->label;
+        $updated_kaltura = $client->baseEntry->update($entry->id, $kaltura_baseEntry);
+      } catch (Exception $ex) {
+        $failureMessage['messages'][0]['message'] = t('Failed to update Kaltura title and description');
```

Comment:
Would it be better to rescind the try and catch statement?

Reply:
I think it is useful to catch and report the error here.  In some cases, the error will be fatal and can be returned directly.  In the case of non-fatal errors (like updating the title and description), we'll want to log the error, but continue processing.  To report the error, it should be returned from the function.  This will mean re-working the existing use of `$failureMessage` to accumulate non-fatal error messages.  In PHP you can add an element to an array as:
```
$failureMessage['messages'][] = $message;
```

The other consideration for this should be: How will the hook caller deal with success or failure messages?  Will it abort processing in any condition?  (I don't think it will, but worth checking.)  If it will abort processing upon receiving a failure message, there is no such thing as a "non-fatal" return.

Reply:
Processes are not terminated upon receiving failure messages, so far the process only terminates if a variable does not exist or an error occurs.